src/libsyntax/feature_gate.rs

```diff
@@ -1136,6 +1136,15 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
                     },
                     _ => {}
                 }
+
+                match defaultness {
```

Comment:
@nagisa Not sure I've understood this. Why should I check for hir::Defaultness in libsyntax?